md.py

```diff
@@ -134,6 +134,13 @@ class MDInvalidArgumentError(Exception):
     pass
 
 class MDInstance:
+    # Raid superblock v1.2 consumes (256 + 2 * ndisks) bytes at a 4k offset
```

Comment:
Comment needs an update with the size change

Reply:
Fixed